lib/engine/step/g_1873/draft.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require_relative '../base'
+
+module Engine
+  module Step
+    module G1873
+      class Draft < Base
+        attr_reader :companies, :choices
+
+        ACTIONS = %w[bid pass].freeze
+        PREMIUM_REDUCTION = 10
+
+        def setup
+          @companies = @game.start_companies.sort
+        end
+
+        def available
+          @companies
+        end
+
+        def may_purchase?(_company)
+          true
+        end
+
+        def auctioning; end
+
+        def bids
+          {}
+        end
+
+        def visible?
+          true
+        end
+
+        def players_visible?
+          true
+        end
+
+        def name
+          'Draft'
+        end
+
+        def description
+          'Draft Private Mines and Concessions'
+        end
+
+        def finished?
+          @companies.empty? || entities.all?(&:passed?) & @game.premium.zero?
```

Comment:
&& ? 

put in parens order is not obvious

Reply:
Yep, typo that sorta works.